src/app/components/budget-display/budget-display.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="transactions">
+  <div class="incomes">
+    @for(income of incomes(); track income){
```

Comment:
The default `track` implementation returns the item itself.
Do you think we can do remove track on this line?
What you have here is not wrong, but I don't see why we need to explicitly set it to income.